var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -105,6 +109,8 @@ class Qt(Package):
     # depends_on("ogg", when='+multimedia')
 
     use_xcode = True
+    # parallel build fails on Sierra
+    parallel = False
```

Comment:
however, `libpng` is external lib, so i don't immediately see what's happening here...